tekton-catalog/pipeline-loops/README.md

```diff
@@ -0,0 +1,32 @@
+# Custom Task: Pipeline Loops
+
+`Pipeline Loops` is implements based on [Tekton Custom Task](https://github.com/tektoncd/community/blob/master/teps/0002-custom-tasks.md) and the template of [Knative sample controller](https://github.com/knative-sandbox/sample-controller)
+
+# Goal
+`Pipeline Loops` is trying to provide a pipeline level loops to handle withItems loop in `Pipeline`(Tekton).
+
+# Take a try
+- Generate the code of client
+  `go mod vendor; go mod tidy; bash hack/update-codegen.sh`
+- Install&config [KO](https://github.com/google/ko)
+- Install Exception Handler
+  `ko apply -f config/`
```

Comment:
can you also add a docker image build so we can run it on our ci and for users who have problem with ko?

Reply:
@ Tomcli I addressed you comments, by adding an dockerfile, makefile, and the readme steps for user to build his own docker file to install the controller